### DISCUSSION TITLE
Where do I get past mission data?

### DISCUSSION THREAD

**Discussion by louis030195** [nasa/openmct#7102] [Community Q&A] [answered]
Thanks 🙏

**Comment by ozyx** [comment, marked as answer]
Hi @louis030195 !

Great question. Check out NASA's [Planetary Data System ](https://pds.nasa.gov/) for data from past NASA planetary missions. You might also have a browse through [NASA's Data Portal](https://data.nasa.gov/) for a general collection of publicly available NASA datasets.

Hope this helps!

Cheers
